datafusion/physical-plan/src/lib.rs

```diff
@@ -116,12 +116,7 @@ pub mod udaf {
 /// [`required_input_ordering`]: ExecutionPlan::required_input_ordering
 pub trait ExecutionPlan: Debug + DisplayAs + Send + Sync {
     /// Short name for the ExecutionPlan, such as 'ParquetExec'.
```

Comment:
Perhaps we can also update the doc string to include a note about using `static_name` if desired to help people migrate

Reply:
Great point 👍 I add an implementation note section in doc string